android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -80,6 +83,8 @@ public class RNSentryModule extends ReactContextBaseJavaModule {
     // 16ms (slower than 60fps) to constitute slow frames.
     private static final int SLOW_FRAME_THRESHOLD = 16;
 
+    private static final int SCREENSHOT_TIMEOUT_SECONDS = 10;
```

Comment:
```suggestion
    private static final int SCREENSHOT_TIMEOUT_SECONDS = 2;
```

H: I believe 10 is too much, you can trigger 2 ANRs with 10s already.